src/cargo/util/toml/mod.rs

```diff
@@ -2122,6 +2148,40 @@ impl TomlManifest {
     }
 }
 
+fn abs_path_for_deps(
```

Comment:
Is this change also left over from the experiment?

Reply:
Yes, I thought we were going ahead with that change so all of it is form the experiment